tests/meteoroid/src/meteoroid/utilities/vault.py

```diff
@@ -0,0 +1,48 @@
+import hvac
+import os
+import pathlib
+from meteoroid import exceptions
+import logging
+
+
+VAULT_ADDR = "https://clotho.broadinstitute.org:8200"
+VAULT_TOKEN = os.environ['VAULT_TOKEN']
+
+
+def get_authenticated_vault_client(token: str) -> hvac.Client:
+    """Returns an authenticated hvac.Client instance."""
+    if pathlib.Path(token).is_file():
+        logging.info(f"Reading the Vault token from file {token}")
```

Comment:
I'm not sure if we want the tests to log the vault token here

Reply:
The token here would be the path to the file instead of the content. But I think it's a potential risk! Changed!